js/electron.js

```diff
@@ -47,7 +47,7 @@ function createWindow() {
 
 	// and load the index.html of the app.
 	//mainWindow.loadURL('file://' + __dirname + '../../index.html');
-	mainWindow.loadURL("http://localhost:" + config.port);
+	mainWindow.loadURL(`http://${config.address}:${config.port}`);
```

Comment:
Back ticks are used because this is a templated string; in other words, you have to use those back ticks otherwise you get a syntax error